Example/src/App.tsx

```diff
@@ -1,332 +1,72 @@
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+import { Button, StyleSheet, View } from 'react-native';
+
 import React from 'react';
```

Comment:
Perhaps you forgot to restore the `App.tsx` file content

Reply:
Yes, indeed. Restored in ac5be7dbdd77a9c91a3ebefb8e22a7590e88b2bc